internal/pkg/metrics/kyma.go

```diff
@@ -141,3 +146,68 @@ func calcStateValue(state, newState shared.State) float64 {
 func (k *KymaMetrics) RecordRequeueReason(kymaRequeueReason KymaRequeueReason, requeueType RequeueType) {
 	k.requeueReasonCounter.WithLabelValues(string(kymaRequeueReason), string(requeueType)).Inc()
 }
+
+func (k *KymaMetrics) CleanupNonExistingKymaCrsMetrics(ctx context.Context, kcpClient client.Client) error {
+	currentLifecycleManagerLogs, err := FetchLifecycleManagerMetrics()
```

Comment:
Intended to name `Logs` and not `Metrics`?

Reply:
No,  thanks for noticing that :D